### DISCUSSION TITLE
The UI looks incredible.

### DISCUSSION THREAD

**Discussion by xidoc** [aleksilassila/reiverr#71] [General]
The UI looks incredible, as does the discovery of new content. 
Why not bring the Jellyfin team on board to integrate all this into Jellyfin, especially as Jellyfin apps are well integrated into other platforms (including connected TVs).

Wouldn't that be a waste of effort ?

If Jellyfin looked like that, I'd happily give up plex for my media videos.

**Comment by aleksilassila** [comment]
Thanks! That would probably be the ultimate solution, but I'm afraid it's probably out of Jellyfin's project scope. Without being too familiar with Jellyfin's team's goals, I'd imagine they are already busy maintaining their current features. Redesigning their whole interface and integrating Radarr & Sonarr would be incredibly time-consuming and probably not practical. Then again, Sonarr and Radarr is something that not all of their users are even interested in.

The good thing is I'm planning on making an android tv release since that's the primary way I consume content myself.

**Reply by rodrigogonegit** [reply]
Doing God's work here
